phpcs.xml

```diff
@@ -342,17 +314,8 @@
   <rule ref="Squiz.WhiteSpace.LanguageConstructSpacing" />
   <rule ref="Squiz.WhiteSpace.SemicolonSpacing"/>
   <rule ref="Squiz.WhiteSpace.SuperfluousWhitespace"/>
-  <rule ref="Squiz.WhiteSpace.OperatorSpacing">
-    <properties>
-      <property name="ignoreNewlines" value="true"/>
-    </properties>
-  </rule>
 
   <!-- Zend sniffs -->
   <rule ref="Zend.Files.ClosingTag"/>
 
-  <!-- sniffs from https://github.com/acquia/coding-standards-php -->
-  <rule ref="SlevomatCodingStandard.Namespaces.AlphabeticallySortedUses" />
```

Comment:
These are from Acquia coding standards and sorts use statements in ascending order. This can be removed as sorting manually is a challenge but phpcbf can help sort it.